src/core/client/admin/routes/Moderate/Queue/ModerationQueueQuery.tsx

```diff
@@ -0,0 +1,47 @@
+import { Match } from "found";
+import React, { FunctionComponent } from "react";
+import { graphql } from "react-relay";
+
+import { QueryRenderer } from "coral-framework/lib/relay";
+
+import {
+  ModerationQueueQuery as QueryTypes,
+  ModerationQueueQueryResponse,
+} from "coral-admin/__generated__/ModerationQueueQuery.graphql";
+
+import ModerationQueue from "./ModerationQueue";
+
+interface Props {
+  settings: ModerationQueueQueryResponse;
+  match: Match;
+}
+
+const ModerationQueueQuery: FunctionComponent<Props> = ({ match }) => {
+  return (
+    <QueryRenderer<QueryTypes>
+      query={graphql`
+        query ModerationQueueQuery {
+          settings {
+            moderation
+          }
+        }
+      `}
+      variables={{}}
+      fetchPolicy="store-and-network"
```

Comment:
This should me `network-only` otherwise it would always try to render twice. Our network layer has a caching mechanism.

Reply:
Never mind! I think `store-and-network` is actually a good compromise.